Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1025,9 +1025,9 @@ public static void HandleHeal(Session session, params string[] parameters)
             session.Player.Stamina.Current = session.Player.Stamina.UnbuffedValue;
             session.Player.Mana.Current = session.Player.Mana.UnbuffedValue;
 
-            var updatePlayersHealth = new GameMessagePrivateUpdateAttribute2ndLevel(session, Vital.Health, session.Player.Health.Current);
-            var updatePlayersStamina = new GameMessagePrivateUpdateAttribute2ndLevel(session, Vital.Stamina, session.Player.Stamina.Current);
-            var updatePlayersMana = new GameMessagePrivateUpdateAttribute2ndLevel(session, Vital.Mana, session.Player.Mana.Current);
```

Comment:
GameMessagePrivateUpdateAttribute2ndLevel is the right message for what this command is doing, I think you've found an error with how we're handling the sequence for this message however. 
![vitalupdates](https://cloud.githubusercontent.com/assets/11369233/25648090/0ff935f4-2f96-11e7-8e20-3ca513bc3200.png)

3 different numbers in a row based on this retail pcap...